101-vs2019-git-docker-windows2019/azuredeploy.json

```diff
@@ -0,0 +1,262 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "storagename": {
+            "type": "string",
+            "metadata": {
+                "description":"Unique Storage Account name"
+            }
+        },
+        "vm-dns": {
+            "type": "string",
+            "metadata": {
+                "description": "VM DNS Label prefix"
+            }
+        },
+        "adminUser": {
+            "type": "string",
+            "defaultValue":"mahendra",
+            "metadata": {
+                "description": "Admin Username for Lab VM"
+            }
+        },
+        "adminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Password for admin user"
+            }
+        },
+        "vmsize": {
+            "type": "string",
+            "defaultValue": "Standard_D8s_v3",
+            "allowedValues": [
+                "Standard_D4s_v3",
+                "Standard_D8s_v3",
+                "Standard_D8s_v4"
+            ],
+            "metadata": {
+                "description": "VM Size for lab-vm"
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Location to deploy current resource"
+            }
+        },
+        "_artifactsLocation": {
+            "type": "string",
+            "metadata": {
+                "description": "The base URI where artifacts required by this template are located including a trailing '/'"
+            },
+            "defaultValue": "[deployment().properties.templateLink.uri]"
+        },
+        "_artifactsLocationSasToken": {
+            "type": "securestring",
+            "defaultValue": "",
+            "metadata": {
+                "description": "SAS Token for accessing script path"
+            }
+        }
+    },
+    "variables":{
+        "scriptUrl": "[uri(parameters('_artifactsLocation'),concat('installscript.ps1',parameters('_artifactsLocationSasToken')))]"
+    },
+    "resources": [    
+        {
+            "name": "[toLower(parameters('storagename'))]",
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties

Reply:
Done, for all resources in template.